README.md

```diff
@@ -58,7 +58,7 @@ Thanks goes to these wonderful people ([emoji key](https://github.com/kentcdodds
-| [<img src="https://avatars3.githubusercontent.com/u/1126497?s=460&v=4" width="100px;"/><br /><sub><b>Kevin Simper</b></sub>](https://github.com/kevinsimper)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=kevinsimper "Code") | [<img src="https://avatars3.githubusercontent.com/u/1572026?s=460&v=4" width="100px;"/><br /><sub><b>Romulo Cintra</b></sub>](https://github.com/romulocintra)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=romulocintra "Code") | [<img src="https://avatars3.githubusercontent.com/u/18748390?s=460&v=4" width="100px;"/><br /><sub><b>Hassan</b></sub>](https://github.com/ichthub)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=ichthub "Code") | [<img src="https://avatars1.githubusercontent.com/u/8556085?s=460&v=4" width="100px;"/><br /><sub><b>Velu S Gautam</b></sub>](https://github.com/ichthub)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=velusgautam "Code")|
+| [<img src="https://avatars3.githubusercontent.com/u/1126497?s=460&v=4" width="100px;"/><br /><sub><b>Kevin Simper</b></sub>](https://github.com/kevinsimper)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=kevinsimper "Code") | [<img src="https://avatars3.githubusercontent.com/u/1572026?s=460&v=4" width="100px;"/><br /><sub><b>Romulo Cintra</b></sub>](https://github.com/romulocintra)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=romulocintra "Code") | [<img src="https://avatars3.githubusercontent.com/u/18748390?s=460&v=4" width="100px;"/><br /><sub><b>Hassan</b></sub>](https://github.com/ichthub)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=ichthub "Code") | [<img src="https://avatars1.githubusercontent.com/u/8556085?s=460&v=4" width="100px;"/><br /><sub><b>Velu S Gautam</b></sub>](https://github.com/velusgautam)<br />[💻](https://github.com/mdn/interactive-examples/commits?author=velusgautam "Code")|
```

Comment:
It's really hard to review this diff, but am I right in thinking that this just corrects the link to your GitHub profile?

https://github.com/ichthub -> https://github.com/velusgautam

?

Reply:
(it's not your fault that the diff is hard to review of course, sorry if that sounded like blaming you)

Reply:
Yes. My Github link was wrong. 😀

Reply:
I'm not sure how this works actually. I wonder if this section of the README is supposed to be generated automatically from https://github.com/mdn/interactive-examples/blob/master/.all-contributorsrc - is that right @schalkneethling ?

Reply:
Essentially yes :) I used to always do this so I unfortunately never documented it. Let me add a to-do to my list to document how to use `all-contributors`. I have been meaning to see how the tool has improved over time anyhow. I hear some 🐦 say this can be largely(if not entirely) automated now.